utils/0001-u-boot-Fix-path-to-merge_config.sh.patch

```diff
@@ -1,9 +1,9 @@
-From afa3f6029bb6c232cde8cae80073b8ee0749b0bb Mon Sep 17 00:00:00 2001
+From 338b578e4c24860d7adf94ac04df4479c1acd52f Mon Sep 17 00:00:00 2001
 From: Ferry Toth <ftoth@exalondelft.nl>
-Date: Wed, 21 Dec 2022 22:39:23 +0100
-Subject: [PATCH 1/1] [PATCH] u-boot: Fix path to merge_config.sh
+Date: Sun, 22 Dec 2024 20:02:46 +0100
+Subject: [PATCH 1/1] u-boot: Fix path to merge_config.sh
 
-Current u-boot recipe in kirkstone fails during do_configure because
+Current u-boot recipe in scarthgap fails during do_configure because
```

Comment:
Hmm... Shouldn't be this fixed in the Yocto Poky? Or did I get this incorrectly?

Reply:
This is a refresh of an old patch we have since 2019. In principle you are right. However, we have our own U-Boot recipe (based on the old Intel Edison one). And Yocto has something newer. In our case configure fails because we do include their newer configure code. I guess they don't have this problem, so we must be doing something else wrong.
Likely the real solution would be to use the upstream recipe to build U-Boot, if needed to use all the Yocto trickery to get the Edison version the same as we have now.
But for now, I'd like to leave this as is.